db.go

```diff
@@ -114,12 +127,23 @@ func (s *ColumnStore) DB(name string) (*DB, error) {
 		highWatermark: atomic.NewUint64(0),
 	}
 
+	/*if s.enablePersistency {
+		// create root database folder
+		if err := os.Mkdir(db.StorePath(), 0777); err != nil && !os.IsExist(err) {
+			return nil, err
+		}
+	}*/
+
 	db.txPool = NewTxPool(db.highWatermark)
 
 	s.dbs[name] = db
 	return db, nil
 }
 
+func (db *DB) StorePath() string {
+	return path.Join(db.columnStore.storagePath, db.name)
```

Comment:
Can we maybe just encapsulate the prefix using a wrapper that implements the `Bucket` interface? Then the table doesn't even need to know anything about the prefix(es). I just checked and unfortunately the objstore package doesn't seem to have that yet, but I'd be happy implementing that wrapper within this repo for now.

Reply:
Yes, it seems a good idea. I'll add a BucketDecorator object